OPConsultation-output.json

```diff
@@ -0,0 +1,2874 @@
+          "type": "Patient"
+        },
+        "recordedDate": "2024-08-06T12:18:11+05:30"
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:e79cb5db-f361-4109-909c-b894a082c456",
+      "resource": {
+        "resourceType": "Condition",
+        "id": "e79cb5db-f361-4109-909c-b894a082c456",
+        "meta": {
+          "versionId": "0",
+          "lastUpdated": "2024-08-06T12:18:11.170+05:30",
+          "profile": [
+            "https://nrces.in/ndhm/fhir/r4/StructureDefinition/Condition"
+          ]
+        },
+        "clinicalStatus": {
+          "coding": [
+            {
+              "code": "resolved",
+              "system": "http://terminology.hl7.org/CodeSystem/condition-clinical",
+              "display": "Resolved"
+            }
+          ],
+          "text": "resolved"
+        },
+        "verificationStatus": {
+          "coding": [
+            {
+              "code": "confirmed",
+              "system": "http://terminology.hl7.org/CodeSystem/condition-ver-status",
+              "display": "Confirmed"
+            }
+          ]
+        },
+        "code": {
+          "coding": [
+            {
+              "code": "53741008",
+              "system": "http://snomed.info/sct",
+              "display": "Coronary artery disease"
+            }
+          ],
+          "text": "This code represents the diagnosis of coronary artery disease, a condition characterized by the narrowing or blockage of the coronary arteries due to atherosclerosis, which can lead to chest pain, heart attacks, and other complications."
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "type": "Patient"
+        },
+        "recordedDate": "2024-08-06T12:18:11+05:30"
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:86d01ed0-b2bd-4b93-8458-28b443e6ceab",
+      "resource": {
+        "resourceType": "Procedure",
+        "id": "86d01ed0-b2bd-4b93-8458-28b443e6ceab",
+        "meta": {
+          "versionId": "0",
+          "lastUpdated": "2024-08-06T12:18:11.173+05:30",
+          "profile": [
+            "https://nrces.in/ndhm/fhir/r4/StructureDefinition/Procedure"
+          ]
+        },
+        "status": "preparation",
+        "code": {
+          "coding": [
+            {
+              "code": "38341003",
+              "system": "http://snomed.info/sct",
+              "display": "High blood pressure"
+            }
+          ],
+          "text": "hypertension"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "type": "Patient"
+        },
+        "performedDateTime": "2020-01-01T10:30:00+05:30",
+        "note": [
+          {
+            "text": "This patient is suffering from Stage 2 Hypertension."
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:4e966b5a-f19a-44b8-943c-00f021198710",
+      "resource": {
+        "resourceType": "Procedure",
```

Comment:
again, wrong procedure code. the display is also wrong

53741008 | Coronary arteriosclerosis (disorder) - is for condition, its not a procedure code.